ssvsigner/cmd/purge-keys/purge-keys.go

```diff
@@ -56,14 +57,18 @@ func run(logger *zap.Logger, cli CLI) error {
 		zap.Bool("client_tls_enabled", cli.Web3SignerKeystoreFile != "" || cli.Web3SignerServerCertFile != ""),
 	)
 
-	if err := bls.Init(bls.BLS12_381); err != nil {
-		return fmt.Errorf("init BLS: %w", err)
+	if cli.BatchSize <= 0 {
+		return fmt.Errorf("invalid batch size %d, must be > 0", cli.BatchSize)
 	}
 
 	if _, err := url.ParseRequestURI(cli.Web3SignerEndpoint); err != nil {
 		return fmt.Errorf("invalid WEB3SIGNER_ENDPOINT format: %w", err)
 	}
 
```

Comment:
I think it is worth separating this into a separate function. `ssv-signer.go` uses a similar pattern

Reply:
fixed